tests/app/dao/notification_dao/test_notification_dao_performance_platform.py

```diff
@@ -13,26 +12,21 @@
 from app.dao.notifications_dao import dao_get_total_notifications_sent_per_day_for_performance_platform
 
 
-BEGINNING_OF_DAY = date(2016, 10, 18)
```

Comment:
All of these tests were marked serial because of these two module constants being used in all tests. Made it impossible to run in parallel so this was a minor refactor to specify non-overlapping dates.